h2o-py/h2o/estimators/estimator_base.py

```diff
@@ -409,3 +409,56 @@ def _metrics_class(model_json):
         else:
             raise NotImplementedError(model_type)
         return [metrics_class, model_class]
+
+    def convert_H2OXGBoostParams_2_XGBoostParams(self, num_class):
+        '''
+        Given a H2OXGBoost model, this method will generate the corresponding parameters that should be used
+        by native XGBoost in order to give exactly the same result assuming that the same dataset (dervied from
+        h2oFrame) is used to train the native XGBoost model.
```

Comment:
Change to:
Given an H2OXGBoost model, this method will generate the corresponding parameters that should be used
by native XGBoost in order to give exactly the same result, assuming that the same dataset (derived from
h2oFrame) is used to train the native XGBoost model.